controllers/home.js

```diff
@@ -1,6 +1,12 @@
 const HomeController = {
   Index: (req, res) => {
-    res.render("home/index", { title: "Acebook" });
+    let username = null;
+    if (req.session.user) {
+      username = req.session.user.username;
+      console.log(username);
+    }
+    console.log(username);
```

Comment:
does this need to be logged twice?

Reply:
@ZacMossHK it doesn't, I have removed one of the logs, should be good to go now